XRTK-Core/Packages/com.xrtk.core/Utilities/Solvers/Solver.cs

```diff
@@ -259,9 +231,9 @@ protected void UpdateTransformToGoal()
                 var rot = cachedTransform.rotation;
                 var scale = cachedTransform.localScale;
 
-                pos = SmoothTo(pos, GoalPosition, SolverHandler.DeltaTime, moveLerpTime);
-                rot = SmoothTo(rot, GoalRotation, SolverHandler.DeltaTime, rotateLerpTime);
-                scale = SmoothTo(scale, GoalScale, SolverHandler.DeltaTime, scaleLerpTime);
+                pos.SmoothTo(GoalPosition, SolverHandler.DeltaTime, moveLerpTime);
```

Comment:
I think all these places where I used the extension method actually broke these as their new values aren't being assigned back.